CHANGELOG.md

```diff
@@ -3,14 +3,8 @@ We follow the [Semantic Versioning 2.0.0](http://semver.org/) format.
 
 ## Unreleased
 
-### Added
-- 
-
 ### Changed
-- 
-
-### Removed
-- 
+- Normalize markdown formatting across usage.md files
```

Comment:
To Chris's point in the chat room: Just prefix this with `[PATCH] **All components:**` to make it clear to the person preparing the release PR that every component needs a bump.